src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: $card-width;
+  border: 1px solid $background-color;
+  border-radius: 5px;
+  transition-duration: .3s;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 32px 16px 16px;
+  }
+
+  &__photo {
+    object-fit: contain;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $text-color;
+    text-decoration: none;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 4px 0 16px;
+    color: $secondary-color;
+  }
+
+  &__rating {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $text-color;
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $text-color;
+  }
+
+  &__buy {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-align: center;
+    width: 166px;
+    height: 40px;
+    background-color: $primary-color;
+    border: 1px solid $primary-color;
+    color: $button-text-color;
+    border-radius: 5px;
+    text-decoration: none;
+    transition-duration: .3s;
```

Comment:
https://codeguide.academy/html-css.html#css-order

Check the same in other places